src/controller_examples/zookeeper_controller/spec/reconciler.rs

```diff
@@ -27,16 +27,17 @@ pub struct ZookeeperReconciler {}
 
 impl Reconciler<ZookeeperClusterView> for ZookeeperReconciler {
     type T = ZookeeperReconcileState;
-    type I = ZKSupportInputView;
-    type O = ZKSupportOutputView;
+    type LibRequest = ZKAPIInputView;
+    type LibResponse = ZKAPIOutputView;
+    type LibState = ZooKeeperState;
```

Comment:
Could you rename them to `ExternalAPIInput`, `ExternalAPIOutput`, and `ExternalState`?